cmd/operator-controller/main.go

```diff
@@ -411,6 +411,7 @@ func run() error {
 		ActionClientGetter:  acg,
 		Preflights:          preflights,
 		BundleToHelmChartFn: convert.RegistryV1ToHelmChart,
+		SystemNamespace:     cfg.systemNamespace,
```

Comment:
Just a tiny question: Why we must you pass the NS now?
I do not see us using it? Is that required so we can delete the release?
If so, all fine 👍 just to confirm. 
I i

Reply:
great spot, this is a leftover from one of the previous implementations, before we made the changes to `helm-operator-plugins` - I'll remove it